dc-cudami-server/dc-cudami-server-business/src/main/java/de/digitalcollections/cudami/server/business/impl/service/legal/LicenseServiceImpl.java

```diff
@@ -66,6 +67,11 @@ public License getByUuid(UUID uuid) {
     return repository.getByUuid(uuid);
   }
 
+  @Override
+  public List<Locale> getLanguages() {
+    return repository.getLanguages();
+  }
```

Comment:
Das könnte auch in die Basis. Ist ja überall dasselbe.

Reply:
Dito :arrow_up: